binding.gyp

```diff
@@ -23,8 +34,9 @@
             "src/windows/functions.cpp"
           ],
           "libraries": [
-            "-luser32.lib",
```

Comment:
@jviotti where did the need for user32.lib come from? Couldn't find a method that required it – is it a leftover from previous iterations, or did I miss something?

Reply:
I don't remember, but it might have been a leftover. If everything works fine without it, then I'm pretty sure we don't need it.